### DISCUSSION TITLE
When we create a tag and put it on the client, it doesn't show on contacts.

### DISCUSSION THREAD

**Discussion by brendewguedes** [chatwoot/chatwoot#7655] [Self hosted]
When we create a tag and put it on the client, it doesn't show on contacts. This is a mistake, since we already add tags during a conversation, and when we go to contacts we must put them again. This delays, moreover at the time of the report it will get in the way. the ideal would be that when we place the labels during a conversation, it is used in the contacts' labels automatically.

**Comment by glauberssoutodossantos** [comment]
This is very important when making a campaign.

**Comment by jtayrone** [comment]
Do it as follows: Add your tag as a custom attribute to the contact, not the conversation, if you want this tag to remain fixed on the contact.

![contact](https://github.com/chatwoot/chatwoot/assets/34479816/27d07fa8-a6d2-4ff9-b0c0-c92f2fa6bef0)

**Reply by cruz4d3r** [reply]
I have this issue too, i try sync the conversations with a PHP code, conect to API and update the contacts with information of conversations.
